test/extended/util/annotate/rules.go

```diff
@@ -182,6 +176,7 @@ var (
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should serve the correct routes when scoped to a single namespace and label set`,
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should set Forwarded headers appropriately`,
 			`\[sig-network\]\[Feature:Router\] The HAProxy router should support reencrypt to services backed by a serving certificate automatically`,
+			`\[sig-network\] Networking should provide Internet connection for containers \[Feature:Networking-IPv6\]`, // ipv6 only clusters do not have external connectivity
```

Comment:
Drop the comment; that's not an intrinsic fact about IPv6, it's just a fact about the current e2e-metal-ipi-ipv6.

(It's correct to add this test to `[Skipped:Disconnected]`, because the test does require external connectivity and thus needs to be skipped if you don't have external connectivity. It's just wrong to claim that the skipping is somehow related to the IPv6-ness rather than the lack-of-external-connectivity-ness.)

Reply:
Yeah, that's not what I meant to say.
Will remove the comment

Reply:
@danwinship done, sorry for the delay